test/Dialect/SV/cse.mlir

```diff
@@ -0,0 +1,12 @@
+// RUN: circt-opt -cse %s | FileCheck %s
+
+// CHECK-LABEL: @no_cse_macro_ref()
+// CHECK: [[VAR0:%.+]] = sv.macro.ref< "PRINTF_COND_"> : i1
+// CHECK: [[VAR1:%.+]] = sv.macro.ref< "PRINTF_COND_"> : i1
+// CHECK: comb.and [[VAR0]], [[VAR1]]
+hw.module @no_cse_macro_ref() -> (out: i1) {
+  %PRINTF_COND__0 = sv.macro.ref< "PRINTF_COND_"> : i1
+  %PRINTF_COND__1 = sv.macro.ref< "PRINTF_COND_"> : i1
```

Comment:
What are the semantics of CSE here?

Reply:
No CSE is allowed, I mention that in the description of the OP.